parachain-template/polkadot-launch/config.json

```diff
@@ -0,0 +1,39 @@
+{
+	"relaychain": {
+		"bin": "../../polkadot/target/release/polkadot",
+		"chain": "rococo-local",
+		"nodes": [
+			{
+				"name": "alice",
+				"wsPort": 9944,
+				"port": 30444
+			},
+			{
+				"name": "bob",
+				"wsPort": 9955,
+				"port": 30555
+			}
+		]
+	},
+	"parachains": [
+		{
+			"bin": "../target/release/parachain-collator",
+			"id": "200",
+			"balance": "1000000000000000000000",
+			"nodes": [
+				{
+					"wsPort": 9988,
+					"name": "alice",
+					"port": 31200,
+					"flags": [
+						"--force-authoring",
+						"--",
+						"--execution=wasm"
```

Comment:
Do we actually need to specify the execution engine?

Reply:
yes, that value is passed to the polkadot full node in the collator node and it is required to make it always follow the latest relay chain logic.

Reply:
Do you know if there's a reason why this value is not the default?

Reply:
No idea, maybe because in the future we want to decouple the polkadot node from the collator.

Reply:
You only need this because we don't use strict spec_version bumps etc. It is also important that you use the same commit in polkadot as you use in your template from polkadot to have the exact same native runtime. 

As we can not ensure this, running everything in wasm is more safe.